android/src/main/res/values/appboy.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <!--- Internal Appboy Usage -->
-  <string name="com_appboy_sdk_flavor">REACT</string>
+  <string name="com_appboy_sdk_flavor" translatable="false">REACT</string>
```

Comment:
lets put the translatable attribute before the name attribute to pattern match what we do elsewhere (e.g in AppboyProject)